### PR TITLE
Update guidelines with info on alt text

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Updated guidelines with info on obligatory alt text for screenshots and diagrams in docs.

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/8464.

### CODE REVIEW

File: guidelines/content-guidelines/02-diagrams.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
 
 For details on how to format diagrams and their elements in Kyma documents, see the particular document sections.
 
+## Reference
+
+Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alt text:
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/02-diagrams.md:16]:
```suggestion
Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alternative text:
```

File: guidelines/content-guidelines/02-diagrams.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
 
 For details on how to format diagrams and their elements in Kyma documents, see the particular document sections.
 
+## Reference
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/02-diagrams.md:14]:
```suggestion
## Alternative text
```

File: guidelines/content-guidelines/07-screenshots.md [hunk 3]
```diff
@@ -5,20 +5,32 @@ title: Screenshots
...
 Follow these basic principles when you place screenshots in your content:
+
 - Do not overuse screenshots and limit visual noise.
 - Do not use directional indicators such as "above" and "below" to refer to screenshots. Instead, include a brief introduction before each screenshot that describes its purpose and any necessary details.
 - Do not include the mouse pointer in your screenshots, unless it shows a function related to the content.
 - Do not include unnecessary elements, such as the browser toolbar. Include only elements that are relevant for the given content.
 
 For details on how to format screenshots and their elements in Kyma documents, see the particular document sections.
 
+## Reference
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/07-screenshots.md:16]:
```suggestion
## Alternative text
```

File: guidelines/content-guidelines/07-screenshots.md [hunk 4]
```diff
@@ -5,20 +5,32 @@ title: Screenshots
...
+
+Always add an alternative text that concisely describes the content or function of the image you are referring to. The alt text:
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/07-screenshots.md:18]:
```suggestion
Always add an alternative text that concisely describes the content or function of the image you are referring to. The alternative text:
```

File: guidelines/content-guidelines/02-diagrams.md [hunk 5]
```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
...
+
+Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alt text:
+
+- Helps to maintain accessibility for every visitor, including people with vision impairments.
+- Appears in place of a diagram if it fails to load.
+- Improves the SEO of the website by enabling crawlers to index the diagram contents better.
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/02-diagrams.md:20]:
```suggestion
- Improves SEO of the website by enabling crawlers to index the diagram contents better.
```

File: guidelines/content-guidelines/07-screenshots.md [hunk 6]
```diff
@@ -5,20 +5,32 @@ title: Screenshots
...
+
+- Helps to maintain accessibility for every visitor, including people with vision impairments.
+- Appears in place of an image if it fails to load.
+- Improves the SEO of the website by enabling crawlers to index the image contents better.
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/07-screenshots.md:22]:
```suggestion
- Improves SEO of the website by enabling crawlers to index the image contents better.
```

File: guidelines/content-guidelines/02-diagrams.md (hunk 1 above)
Comment (kazydek) [reply guidelines/content-guidelines/02-diagrams.md:16]: Alt text is another name for alternative text. As a compromise, I can introduce it first in parentheses, is that ok?

File: guidelines/content-guidelines/02-diagrams.md (hunk 2 above)
Comment (kazydek) [reply guidelines/content-guidelines/02-diagrams.md:14]: or Descriptions?

File: guidelines/content-guidelines/07-screenshots.md (hunk 3 above)
Comment (kazydek) [reply guidelines/content-guidelines/07-screenshots.md:16]: as above

File: guidelines/content-guidelines/07-screenshots.md (hunk 4 above)
Comment (kazydek) [reply guidelines/content-guidelines/07-screenshots.md:18]: as above

File: guidelines/content-guidelines/02-diagrams.md (hunk 5 above)
Comment (kazydek) [reply guidelines/content-guidelines/02-diagrams.md:20]: I would stick to `the SEO` - it has move search results on UK and US sites.

File: guidelines/content-guidelines/07-screenshots.md (hunk 6 above)
Comment (kazydek) [reply guidelines/content-guidelines/07-screenshots.md:22]: as above

File: guidelines/content-guidelines/02-diagrams.md (hunk 5 above)
Comment (klaudiagrz) [reply guidelines/content-guidelines/02-diagrams.md:20]: I also checked that and I've mostly seen results with no article. Check out this [wiki article](https://en.wikipedia.org/wiki/Search_engine_optimization) about SEO for example - here SEO also appears with no article.

File: guidelines/content-guidelines/02-diagrams.md [hunk 7]
```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
 
 For details on how to format diagrams and their elements in Kyma documents, see the particular document sections.
 
+## Descriptions
+
+Always add an alternative (alt) texts that concisely describes the content or function of the diagram you are referring to. The alt text:
```
Comment (klaudiagrz) [inline guidelines/content-guidelines/02-diagrams.md]:
```suggestion
Always add an alternative (alt) text that concisely describes the content or function of the diagram you are referring to. The alt text:
```

**Pull request merged**